lib/logstash/inputs/LogStashRecordProcessor.rb

```diff
@@ -0,0 +1,65 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require "java"
+begin
```

Comment:
I would recommend having this ```require_jars``` in a separate and specific file, like this it would simplify version updates, specially as you can see in this file plus in the ```dynamodb.rb``` one where you're requiring the same jars.